spec/spec_helper.rb

```diff
@@ -5,7 +5,14 @@
 
 CENTOS_7 = {
   platform: 'centos',
-  version: '7.2.1511',
+  version: '7.4.1708',
+  log_level: :fatal,
+}.freeze
+
+CENTOS_6 = {
+  platform: 'centos',
+  version: '6.9',
+  log_level: :fatal,
```

Comment:
By the way, when this block is present at the bottom, I don't think you need to specify `log_level` for each platform. Could you remove those lines?

```ruby
RSpec.configure do |config|
  config.log_level = :fatal
end
```

Reply:
Ah yes, we should update the description on the trello card to deal with that.

Reply:
Ok, I updated the Trello card